### PR TITLE
feat: add comment APIs for story

### PR DESCRIPTION
**Pull request by VVVENIIDIICI**
## Changes
- [x] Implement story comment APIs
- [x] Reorganize app.py file
- [ ] Unit tests will be added in later PR. Ideally, I want to start a Mock DB for testing. I still need to figure out how it works.

## How Has This Been Tested?
- Passed `pytest` locally.
- Manually use postman to test the APIs.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/story/comment.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+            comment_content: str, the content of comment(in request body)
+        Return:
+            A response specifying whether the comment creation is successed.
+        """
+        args = parser.parse_args()
+        if args["story_id"] is None or args["user_id"] is None:
+            return generate_response("Error: Invalid parameters.", 400)
+        if args["user_id"] != str(user_id):
+            return generate_response("Error: Not Authorized.", 403)
+        request_data = json.loads(request.get_data())
```
Comment (yifeili98) [inline src/story/comment.py:81]: we get error `json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)` if there is no request body, should we add a error response handler for that case?

**Review by yifeili98 — APPROVED** [review]
LGTM, I will add story APIs ASAP, and then we can develop more filtering stuff, add some dummy stories/comments to google cloud server and sync up frontend/backend

**Comment by yifeili98** [comment]
Thanks for the change!

File: src/story/comment.py (hunk 1 above)
Comment (VVVENIIDIICI) [reply src/story/comment.py]: Fixed. Add an exception handler for the error request body. Thanks for the suggestion!

**Comment by VVVENIIDIICI** [comment]
> LGTM, I will add story APIs ASAP, and then we can develop more filtering stuff, add some dummy stories/comments to google cloud server and sync up frontend/backend

Sounds good. I am looking forward to the PR.

**Pull request merged**